db/migrate/20150113021509_change_email_favorites_in_user.rb

```diff
@@ -0,0 +1,8 @@
+require_relative '20150111155134_add_email_permission_to_users.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.